File: 351004/Student/Lab6/publisher/src/main/java/com/example/task310/dto/LoginRequestTo.java
package com.example.task310.dto;

import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class LoginRequestTo {
    @Size(min = 2, max = 64)
    private String login;
    @Size(min = 8, max = 128)
    private String password;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/dto/request/LoginRequestTo.kt
package com.example.distcomp.dto.request

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class LoginRequestTo(
    @field:NotBlank
    @field:Size(min = 2, max = 64)
    val login: String,
    @field:NotBlank
    @field:Size(min = 8, max = 128)
    val password: String
)
